kind/README.md

```diff
@@ -0,0 +1,25 @@
+<!--TODO(bentheelder): fill this in much more thoroughly-->
```

Comment:
this and `docs/` are missing a lot. I have more, but I'm keeping that for follow-ups. this PR is already size/XL and some of it makes more sense once the functionality exists in code as well